### ISSUE TITLE
Train

### ISSUE THREAD

**Issue by billie2jean** [saqib736/Depth_Estimation#1]
Hi, this is Billie, thanks for your wonderful work!
I want to train the model on my own dataset for Marigold, but I don't know how to change the path of dataset.

**Comment by nmeegan** [comment]
Hi Billie, I have also been playing around with this codebase in order to finetune Marigold. I found that you will need to create a text file for the dataset similar to the format of the existing text file under stable_difusion/filenames for your train and test set (RGB image path first, depth path second). Then, under stable_difusion/train_sd.py you will need to edit the default arguments for dataset_path and train_list/test_list for the training text file and test text file, respectively (or alternatively, add them as flags when running train_sd.py). Let me know if that works for you!

**Comment by saqib736** [comment]
> Hi, this is Billie, thanks for your wonderful work! I want to train the model on my own dataset for Marigold, but I don't know how to change the path of dataset.

You need to put your training data in a folder under the working directory such that the folder contains the RGB folder and the corresponding depth maps folder and then generate the filenames for the dataset. Finally specify the folder path nd filename path in the script

**Comment by billie2jean** [comment]
> Hi Billie, I have also been playing around with this codebase in order to finetune Marigold. I found that you will need to create a text file for the dataset similar to the format of the existing text file under stable_difusion/filenames for your train and test set (RGB image path first, depth path second). Then, under stable_difusion/train_sd.py you will need to edit the default arguments for dataset_path and train_list/test_list for the training text file and test text file, respectively (or alternatively, add them as flags when running train_sd.py). Let me know if that works for you!

Thanks! I'm trying. Hope the problem can be solved smoothly.

**Comment by YacineDeghaies** [comment]
@billie2jean  were you able to train the Model ?

**Comment by billie2jean** [comment]
I can, but I think that's not what I need.

**Comment by YacineDeghaies** [comment]
add me on discord: xarda5
do you actually need metric depth ?

**Comment by saqib736** [comment]
> add me on discord: xarda5 do you actually need metric depth ?

I have updated the stable diffusion directory, now you will be able to train and perform inference easily.
